src/ui/transforms.js

```diff
@@ -40,40 +40,42 @@ https://raw.githubusercontent.com/fluid-project/sjrk-story-telling/master/LICENS
     };
 
     /* A transform to turn an array into a delimited string
-     * Values can be optionally be
-     * included only if they are string value and are truthy (i.e. not an empty
-     * string). Values can also be accessed via a specific path relative to each
-     * term.
-     * - separator: the string delimiter to be inserted between each term
-     * - stringOnly (optional): flag to exclude all falsy or non-string values
-     * - path (optional): an EL path on each item in the terms collection
+     * Values can also be accessed via a specific object path relative to each term.
+     * - "separator" (optional): the delimiter to be inserted between each term
+     * - "trailingSeparator" (optional): flag to include the separator at the end
+     * - "stringOnly" (optional): flag to allow only non-empty strings
+     * - "path" (optional): an EL path on each item in the terms collection
      */
     fluid.defaults("sjrk.storyTelling.transforms.arrayToString", {
         "gradeNames": [ "fluid.standardTransformFunction", "fluid.multiInputTransformFunction" ],
         "inputVariables": {
             "separator": ", ",
-            "stringOnly": "",
+            "trailingSeparator": false,
+            "stringOnly": true,
             "path": ""
         }
     });
```

Comment:
You might as well add an inverse method as well. And can you file a JIRA about contributing these up to Infusion.

Reply:
Can you elaborate on what you mean by an inverse method? Would the transformation above it (stringToArray) fit the bill, or do you mean something within arrayToString that indicates what its inverse is?

Reply:
Yes, that function would probably work. You'd need to setup the `invertConfiguration` option in the transformation grades.